ibc/utils.go

```diff
@@ -163,6 +177,8 @@ func DeriveDecimalsFromDenom(baseDenom string) (uint8, error) {
 	switch baseDenom[0] {
 	case 'u': // micro (u) -> 6 decimals
 		decimals = 6
+	case 'n': // nano (n) -> 9 decimals
+		decimals = 9
```

Comment:
Can't be exploited this approach? I could create a token with 9 decimals but use 'u' in the denom. In this way, when I move the token to Evmos it worth more than the original.

Reply:
can be modified via the proposal